### ISSUE TITLE
ntials, and paste them into the plugin.  ly

### ISSUE THREAD

**Issue by earldgrayjr** [chrisjshull/homebridge-nest#624]
**Describe the bug**
Plugin says Google is logged out after 1-hour use with the Nest page and sidecar still open and logged in.  To make the plugin functional again, I must log out and log back in to the page and get new credentials to paste and save.  Then it just does it again (apparently) after about an hour

**To Reproduce**
1. Verify authentication erroR in Hoobs
2. Open nest.com
3. log out
4. log in
5. Copy credentials
6. paste credentials to the plugin
7. Verify authentication in logs
8. Verify operation in Hoobs
9. Remove accessory in Homekit
10. Add accessory from hoobs in Homekit
11. Verify operation
12. Leave Nest page up and logged in.
13. Verify Google is logged in

**Expected behavior**
Expect plugin to work in hoobs and homekit continuously.  Not sure if this is a Nest or plugin problem, but pretty unusable currently

![hoobs nest](https://github.com/chrisjshull/homebridge-nest/assets/14880354/b55eed53-3689-4cec-92fa-8c4169bbb505)
url)

**Comment by Skates1616** [comment]
I am running into the same issue, the access token is not changing, but the cookie is just changing ever so slightly everytime I open home.nest.com.

**Comment by earldgrayjr** [comment]
Mine works, but only for a short time.  Nest says its Hoobs.  Hoods tells
me to shut down the page after getting token/cookies (which I explained I
had already done).
Then it very reliably quits working, while the log shows an authentication
error, and says to make sure I haven't logged out of Google (which I
haven't).
At that point it just doesn't work, so I try to restart the plug-in, the
same result. Restart Hoobs, the same result.
I go back to the nest.home.com page, and it says the token/cookies have
expired and instructs me to log off and log back on again, where it gives
me a new token/cookies, and I am in groundhof=g day again.
 I have tried uninstalling the plug-in and starting from scratch but also
got the same results.  About ready to scrap the whole thing and try
something else.
